tests/stub/txparameters.py

```diff
@@ -42,7 +42,7 @@
 !: AUTO GOODBYE
 !: AUTO RESET
 
-C: BEGIN {"bookmarks": ["b1", "b2"]}
+C: BEGIN {"bookmarks{}": ["b1", "b2"]}
```

Comment:
is it intentional ?

Reply:
Yes. It allows the bookmarks to be sent in arbitrary order.